src/main/java/org/opentripplanner/routing/api/request/preference/TransitPreferences.java

```diff
@@ -120,10 +120,6 @@ public int otherThanPreferredRoutesPenalty() {
     return otherThanPreferredRoutesPenalty;
   }
 
-  /**
-   * A cost function used to calculate penalty for an unpreferred route. Function should return
-   * number of seconds that we are willing to wait for preferred route.
-   */
```

Comment:
We were supposed to remove the javadoc when we add the documentation to the request mapper, right?

Reply:
Can you add a link here:
```
/** 
 * See {@link RoutingRequestMapper}
 */
```